hack/build/etcd-backup-operator/build

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+function go_build {
+	echo "building "${1}"..."
+	if [ ! -z ${GOINSTALL+x} ] && [ "${GOINSTALL}" = "y" ]
+	then
+  		GOBIN=${bin_dir} GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go install -installsuffix cgo  -ldflags "$go_ldflags" ./cmd/${1}/
+		mv ${bin_dir}/${1} ${bin_dir}/etcd-backup-${1}
+	else
+  		GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o ${bin_dir}/etcd-${1} -installsuffix cgo  -ldflags "$go_ldflags" ./cmd/${1}/
+	fi
+}
+
+if ! which go > /dev/null; then
+	echo "golang needs to be installed"
+	exit 1
+fi
+
+GIT_SHA=`git rev-parse --short HEAD || echo "GitNotFound"`
+
+gitHash="github.com/coreos/etcd-operator/version/etcd-backup-operator.GitSHA=${GIT_SHA}"
+
+go_ldflags="-X ${gitHash}"
+
+bin_dir="$(pwd)/_backup_output/bin"
+mkdir -p ${bin_dir} || true
+
+go_build backup-operator
```

Comment:
Just pass in "etcd-backup-operator" ?

Reply:
`cmd/backup-operator` is the folder. we can change this when we change the `cmd/backup-operator` -> `cmd/etcd-backup-operator`

Reply:
I see. Please also change these two lines:
```
mv ${bin_dir}/${1} ${bin_dir}/etcd-backup-${1}
...
go build -o ${bin_dir}/etcd-${1}
```
to get rid of the "etcd-backup-" and "etcd-" prefix first.

Note that we would change this in the future:
```
cmd/backup-operator -> cmd/etcd-backup-operator
```

Reply:
do that in a different pr when this gets merged.

Reply:
It's either to make the change to `cmd/etcd-backup-operator` in this PR, or make naming consistent in this PR and change them in another PR.

Reply:
i'll do `change them together in another PR.`